src/api/database/misc.rst

```diff
@@ -59,11 +41,14 @@
     :<json object: Mapping of document ID to list of revisions to purge
     :>header Content-Type: - :mimetype:`application/json`
                            - :mimetype:`text/plain; charset=utf-8`
-    :>json number purge_seq: Purge sequence number
+    :>json string purge_seq: Purge sequence string
     :>json object purged: Mapping of document ID to list of purged revisions
-    :code 200: Request completed successfully
+    :code 201: Request completed successfully
+    :code 202: Request was accepted, and at least completed successfully on one
+                of the replicas, but the quorum was not reached
```

Comment:
Suggest rewording, for clarity:  "Request was accepted, and was completed successfully on at least one replica, but quorum was not reached."